vapi/simulator/simulator.go

```diff
@@ -291,6 +291,21 @@ func (s *handler) DetachTag(id vim.ManagedObjectReference, tag vim.VslmTagEntry)
 	return nil
 }
 
+// StatusCustom responds with custom HTTP status and json encoded val if given.
+// For use with "/api" endpoints.
+func StatusCustom(w http.ResponseWriter, httpStatus int, val ...interface{}) {
```

Comment:
Could just name this `Status` and change `StatusOK` to use it:
```go
func StatusOK(w http.ResponseWriter, val ...interface{}) {
	Status(w, http.StatusOK, val...)
}
```

Reply:
yeah good catch! didn't wanted to mess with already working things :D changed here